week1/homework/src/server.js

```diff
@@ -10,6 +10,35 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        response.setHeader('Content-Type', 'application/json');
+        response.write(JSON.stringify({ state: state }));
+        break;
+
+      case '/add':
+        state += 1;
```

Comment:
nice that you separated app logic (addition) from other technical details (response.write) 👍